org.eclipse.xtext.xbase.lib.tests/src/org/eclipse/xtext/xbase/tests/lib/ArrayExtensionsBooleanTest.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) 2019, 2020 itemis AG (http://www.itemis.eu) and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.xtext.xbase.tests.lib;
+
+import java.util.Objects;
+
+import org.eclipse.xtext.xbase.lib.ArrayExtensions;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * @author Mathias Rieder - Initial contribution and API
+ */
+public class ArrayExtensionsBooleanTest {
+	private boolean[] data;
+
+	@Before
+	public void init() {
+		data = createData();
+	}
+
+	private boolean[] createData() {
+		return new boolean[] {true, false, true};
+	}
+
+	@Test
+	public void testSetGet__boolean() {
+		Assert.assertTrue(data[0]);
```

Comment:
To be honest, since we do not call any of the extension methods in this class and since we don't use the Xbase compiler anymore, this class only tests a boolean array from Java.